.github/workflows/test.sh

```diff
@@ -13,7 +13,26 @@ prun cd example_pkg
 prun spin --version
 
 pip install meson-python ninja
-prun spin build
+
+
+# Test spin build + debug builds
+echo -e "${MAGENTA}Creating debug builds${NORMAL}"
+prun spin build --gcov
+echo -e "${MAGENTA}Did the build folder get generated?${NORMAL}"
+if [ ! -d "build" ] || [ ! -d "build-install" ]; then
+    echo -e "${RED}build and/or build-install folders did not get generated${NORMAL}"
```

Comment:
Makes me think we should also add `perror`!

Reply:
Haha, I was actually thinking about the same thing but didn't want to introduce too many side changes. I'd be happy to refine it 👍

Reply:
🙏

Reply:
Added via [02e9e63](https://github.com/scientific-python/spin/pull/146/commits/02e9e6356ec55708210fa8702b410adc21a9ed02)